app/views/snack/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= @snack.name %></h1>
+<h2><%= @snack.price %></h2>
+<section class="snack-locations">
+  <h3>Available at:</h3>
+  <% @snack.machines.each do |machine| %>
+    <p><%= machine.location %> (<%= machine.snacks.count %> kinds of snacks, average price of $<%= machine.average_snack_price %>) /p>
```

Comment:
Oups!! i totataly put an AR call in the view here `.count` :(

Reply:
😏 Good catch though

Reply:
😏 Good catch though